cli/cmd/project/connect.go

```diff
@@ -127,6 +129,11 @@ func ConnectCmd(cfg *config.Config) *cobra.Command {
 				prodBranch = ghRes.DefaultBranch
 			}
 
+			envs, err := local.ParseEnvs(envVariables)
```

Comment:
Importing the `local` package here could be quite confusing for a new dev. It's probably best to move this func to a new/another util package in `cli/pkg`